java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -70,4 +82,239 @@ public void customCommand_interruptedException() throws ExecutionException, Inte
+        Map testPayload = new HashMap<String, String>();
+        testPayload.put("key1", "value1");
+        testPayload.put("key2", "value2");
+        testPayload.put("key3", "value3");
+        when(testResponse.get()).thenReturn(testPayload);
+        when(commandManager.<Map>submitNewCommand(eq(cmd), any())).thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<Map> response = service.info();
+        Map payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(testPayload, payload);
+    }
+
+    @Test
+    public void infoWithMultipleOptions_success() throws ExecutionException, InterruptedException {
+        // setup
+        Command cmd =
+                Command.builder()
+                        .requestType(Command.RequestType.INFO)
+                        .arguments(new String[] {"ALL", "DEFAULT"})
+                        .build();
+        CompletableFuture<Map> testResponse = mock(CompletableFuture.class);
+        Map testPayload = new HashMap<String, String>();
+        testPayload.put("key1", "value1");
+        testPayload.put("key2", "value2");
+        testPayload.put("key3", "value3");
+        when(testResponse.get()).thenReturn(testPayload);
+        when(commandManager.<Map>submitNewCommand(eq(cmd), any())).thenReturn(testResponse);
+
+        // exercise
+        InfoOptions options =
+                InfoOptions.builder()
+                        .section(InfoOptions.Section.ALL)
+                        .section(InfoOptions.Section.DEFAULT)
+                        .build();
+        CompletableFuture<Map> response = service.info(options);
+        Map payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(testPayload, payload);
+    }
+
+    @Test
+    public void infoEmptyWithOptions_success() throws ExecutionException, InterruptedException {
+        // setup
+        Command cmd =
+                Command.builder().requestType(Command.RequestType.INFO).arguments(new String[] {}).build();
+        CompletableFuture<Map> testResponse = mock(CompletableFuture.class);
+        Map testPayload = new HashMap<String, String>();
+        testPayload.put("key1", "value1");
+        testPayload.put("key2", "value2");
+        testPayload.put("key3", "value3");
+        when(testResponse.get()).thenReturn(testPayload);
+        when(commandManager.<Map>submitNewCommand(eq(cmd), any())).thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<Map> response = service.info(InfoOptions.builder().build());
+        Map payload = response.get();
+
+        // verify
+        assertEquals(testResponse, response);
+        assertEquals(testPayload, payload);
+    }
+
+    @Test
+    public void get_success() throws ExecutionException, InterruptedException {
+        // setup
+        // TODO: randomize keys
```

Comment:
Does the todo here and in set_success need to be addressed still? Or can we remove the todo?